tests/pipelines/bicep-build-to-validate.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           git_diff1=$(git diff --name-only HEAD^ HEAD infra-as-code/bicep/modules/managementGroups/managementGroups.bicep)
           git_diff2=$(git diff --name-only HEAD^ HEAD infra-as-code/bicep/modules/customRoleDefinitions/customRoleDefinitions.bicep)
           git_diff3=$(git diff --name-only HEAD^ HEAD infra-as-code/bicep/modules/policy/definitions/custom-policy-definitions.bicep)
-          git_diff4=$(git diff --name-only HEAD^ HEAD infra-as-code/bicep/modules/policy/assignments/policyAssignmentManagementGroup.bicep)
+          git_diff4=$(git diff --name-only HEAD^ HEAD infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep)
```

Comment:
Should we also check for changes to the `policyAssignmentManagementGroup.bicep` module as this is used by the `alzDefaultPolicyAssignments.bicep` module multiple times?

Also should we check for changes to policy JSON files as well to trigger this? Or shall we create a separate pipeline we can trigger for these when we want to to just test policy deployment and assignment to the MG hierarchy? Something like `/azp run policy-check`?

Reply:
+1 on the policyAssignmentManagementGroup.bicep. For the individual policy files, I think that one loops into another conversation around policies that we are currently having and so would be outside the scope of this pipeline.